google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -56,17 +56,29 @@ public static void afterClass() throws Exception {
   /**
    * Creates an equality expression for logging filter.
    *
-   * @see <a href="https://cloud.google.com/logging/docs/view/advanced_filters">Advanced Logs
+   * @see <a href= "https://cloud.google.com/logging/docs/view/advanced_filters">Advanced Logs
    *     Filters Documentation</a>
    */
   protected static <V> String createEqualityFilter(String name, V value) {
     return name + "=" + "\"" + value.toString() + "\"";
   }
 
+  protected static boolean cleanupLog(String logName) throws InterruptedException {
+    int deleteAttempts = 0;
+    int allowedDeleteAttempts = 5;
+    boolean deleted = false;
+    while (!deleted && deleteAttempts < allowedDeleteAttempts) {
+      Thread.sleep(5000);
```

Comment:
add logs so when looking at the test run logs we can see how many attempts it took to clean and whether cleanup succeeded or not.

Reply:
I will do it later. Need review what and how it is better done, unless you are sure that a plain `System.out.println()` call will be good enough. I just moved the existing code into the base class. Adding this info is useful but might have few gotchas.